flask_praetorian/base.py

```diff
@@ -722,24 +723,27 @@ def read_token_from_cookie(self):
 
     def read_token(self):
         """
-        Tries to unpack the token from the current flask request in the locations configured with JWT_PLACES
+        Tries to unpack the token from the current flask request
+        in the locations configured by JWT_PLACES.
         Check-Order is defined by the value order in JWT_PLACES.
         """
         for place in self.jwt_places:
             try:
-                return getattr(self, "read_token_from_{place}".format(place=place.lower()))()
+                return getattr(self, "read_token_from_{place}"
```

Comment:
When wrapping argument lines, please put the first argument on a new-line

Reply:
I will adjust it to your wrapping scheme. Line 749 and 740 are longer than 79 characters. Would you prefer this

```
#And what about making flake8 ignore the length of the line?
raise MissingToken(
            "Could not find token in any of the given locations: {locations}".format(  # noqa: E501
                locations=self.jwt_places)
                )
```
or this ?
``` 
raise MissingToken(
            "Could not find token in any of the "
	    "given locations: {locations}".format( 
                locations=self.jwt_places)
                )
```

Reply:
In the next minor version of flask-praetorian, I will be using black as a code formatter. My preferred style is:
```python
raise MissingToken(
    "Could not find token in any of the given locations: {locations}".format(
        locations=self.jwt_places,
    )
)
```